### PR TITLE
DX-1680 - Add clockwise rotation button

### PR DESCRIPTION
**Pull request by corbinschwartz**

### CODE REVIEW

File: src/arrow-clockwise.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
...5.632243,-0.0457302545 132.660347,1.68727242 131.049624,4.53632598 C129.438901,7.38537955 129.438901,10.90908 131.049624,13.7581336 C132.660347,16.6071871 135.632243,18.3401898 138.827316,18.293541 C196.103434,18.3392725 242.280896,65.9976515 242.280896,125.000414 C242.280896,184.03223 196.049981,231.707378 138.734881,231.707378 C95.0218512,231.707378 57.7302246,203.933689 42.479687,164.634429 L60.6600785,175.5911 C63.4065202,177.433599 66.913521,177.561627 69.7791684,175.924006 C72.6448157,174.286386 74.4016057,171.150279 74.347267,167.769334 C74.2929283,164.388389 72.4363258,161.314177 69.5196305,159.775603 L34.0814223,138.434211 C31.9989011,137.186056 29.5139917,136.866708 27.1986114,137.549665 C24.883231,138.232623 22.9368351,139.859054 21.8073988,142.054625 L1.13511039,181.688641 C-0.424336395,184.55503 -0.37386504,188.064891 1.26731299,190.882228 C2.90849101,193.699565 5.8879161,195.390991 9.07149713,195.312681 C12.3434337,195.265759 15.3239616,193.36095 16.8236055,190.358429 L26.3293142,172.161233 C44.4128413,217.862301 88.0044953,250 138.734881,250 L138.734881,250 Z" transform="translate(130.000000, 125.000000) rotate(180.000000) translate(-130.000000, -125.000000)"></path>
```
Comment (hellatan) [inline src/arrow-clockwise.svg]: what happens if the `transform` property gets removed?

Comment (corbinschwartz) [inline src/arrow-clockwise.svg]: https://i.imgur.com/tp5ih3H.png

Comment (corbinschwartz) [inline src/arrow-clockwise.svg]: with it, https://i.imgur.com/h825DQM.png

Comment (hellatan) [inline src/arrow-clockwise.svg]: can the designer have it defaulted to the transformed position without the `transform` attribute? Is that possible? I'd prefer to leave out as many transformations as possible.

Comment (corbinschwartz) [inline src/arrow-clockwise.svg]: I'll ask

**Comment by corbinschwartz** [comment]
@hellatan updated the svg, no longer uses transform

File: src/arrow-clockwise.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 260 250" class="dibsvg-svg dibsvg-arrow-clockwise">
+    <path d="M121.3,0C54.4,0,0,56.1,0,125c0,68.9,54.3,124.9,121.2,125c3.2,0,6.2-1.7,7.8-4.5c1.6-2.8,1.6-6.4,0-9.2
```
Comment (hellatan) [inline src/arrow-clockwise.svg]: nice. did you do that or the designer?

Comment (corbinschwartz) [inline src/arrow-clockwise.svg]: The designer hooked it up, I don't have Illustrator or I'd have taken a stab at it

**Comment by jrdrg** [comment]
+ dibs-vg@1.33.0

**Pull request merged**